<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeAdd</class>
 <widget class="QDialog" name="RecipeAdd">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tytu≈Ç:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Opis:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
